talent/open_source_days.md

```diff
@@ -2,7 +2,7 @@
 # Open Source Days
 
 Developers at source{d} dedicate 10% of their development time to Open Source projects of their choice.
-This is accomplished with Open Source Days (OSD), which happen on the first Monday of each sprint.
+This is accomplished with Open Source Days (OSD), which happen every two Mondays.
```

Comment:
every second Monday?

Reply:
After discussion with @dpordomingo  and @carlosms , what about: ...`every two weeks, on Mondays`? We are not sure which actually is the correct and sounds better.